libraries/AP_Mission/AP_Mission.h

```diff
@@ -269,6 +269,7 @@ class AP_Mission
 
     // MAV_CMD_IMAGE_START_CAPTURE support
     struct PACKED image_start_capture_Command {
+        uint8_t instance;
```

Comment:
I believe this will lead to corruption of the image-start-capture mission command for users that already have the command in a mission on a vehicle.  I think this is OK though because Mission Planner at least never had the columns displayed to users so most users were probably leaving all columns blank.

@peterbarker what do you think?

Reply:
is there any way we can add support without leading corruption ?

Reply:
@khanasif786,

Yes, I think if the instance was placed at the end of the structure instead of the beginning.